root/static/scripts/common/components/EntityLink.js

```diff
@@ -74,13 +74,23 @@ const iconClassPicker = {
 };
 
 const Comment = ({
+  alias,
   className,
   comment,
-}: {+className: string, +comment: string}) => (
+}: {+alias?: string, +className: string, +comment: string}) => (
   <>
     {' '}
     <span className={className}>
-      {bracketed(<bdi key="comment">{comment}</bdi>)}
+      {bracketed(
+        <bdi key="comment">
+          {nonEmpty(alias) ? (
+            <i title={l('Primary alias')}>
+              {alias + (nonEmpty(comment) ? ', ' : '')}
+            </i>
+          ) : null}
```

Comment:
Just realized this should probably be using `commaOnlyList`:

```suggestion
          {nonEmpty(aliasElement) ? (
            nonEmpty(comment) ? (
              commaOnlyList([aliasElement, comment])
            ) : aliasElement
          ) : comment}
```

Where `aliasElement` can be defined above, like:

```js
const aliasElement = nonEmpty(alias)
  ? <i title={l('Primary alias')}>{alias}</i>
  : null;
```

However it also requires changing the component definition to use `=> { ... }` instead of `=> ( ... )`.

Reply:
Thanks, done. I'd considered this earlier (@yvanzo gave the same suggestion), but I think I had worried that it'd look strange to have the comma and space in the non-italicized text node. Looks fine to me, though.